ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml

```diff
@@ -31,37 +31,45 @@ tests:
     test:
     - ref: openshift-extended-rosacli-tests
     workflow: rosa-aws-sts-rosacli-e2e
-- as: aws-rosa-non-sts-advanced-f3
+- as: aws-rosa-non-sts-advanced-rosacli-e2e-f3
```

Comment:
```
-  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-advanced-f3
+  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-advanced-rosacli-e2e-f3
```
The job name already contains `rosacli` ( comes from the config file name). Perhaps no need to duplicate that ?

Reply:
We could have multiple job configures for rosacli-e2e and ocp-e2e, `aws-rosa-sts-advanced-rosacli-e2e` should easily for us to separate from the ocp one `aws-rosa-sts-advanced` .

Reply:
Just rethinking, agree with you. Let us remove `rosacli-e2e` from the job's name.